SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/PropertyValidator.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.Generic;
+
+namespace Microsoft.AppCenter.Windows.Shared.Utils
+{
+    public static class PropertyValidator
+    {
+        private const int MaxProperties = 20;
+        private const int MaxPropertyKeyLength = 125;
+        private const int MaxPropertyValueLength = 125;
+
+        /// <summary>
+        /// Validates properties.
+        /// </summary>
+        /// <param name="properties">Properties collection to validate.</param>
+        /// <param name="logName">Log name.</param>
+        /// <returns>Valid properties collection with maximum size of 5</returns>
```

Comment:
`<returns>Valid properties collection with maximum size of 5</returns>`

Is this true? Or should it be `with maximum size of {MaxProperties}`?

Reply:
We need to update swagger then